apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationStatement.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react'
+import django from 'django'
+
+export const ModerationStatement = (props) => {
+  const [statement, setStatement] = useState(props.initialStatement)
```

Comment:
in case `props.initialStatement` is `undefined` you might need to initialize `statement` with an empty string. otherwise the JSX below `statement.length` might break.

maybe something like `useState(props.initialStatement | '')`

Reply:
ahh nice, I initially had it defined within the props parentheses but didn't like that and wasn't sure where else to put it, thank you!